rules/CodeQuality/Rector/Property/TypedPropertyFromToOneRelationTypeRector.php

```diff
@@ -23,14 +24,19 @@
 use Rector\ValueObject\PhpVersion;
 use Rector\ValueObject\PhpVersionFeature;
 use Rector\VersionBonding\Contract\MinPhpVersionInterface;
-use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\ConfiguredCodeSample;
 use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+use Webmozart\Assert\Assert;
 
 /**
  * @see \Rector\Doctrine\Tests\CodeQuality\Rector\Property\TypedPropertyFromToOneRelationTypeRector\TypedPropertyFromToOneRelationTypeRectorTest
  */
-final class TypedPropertyFromToOneRelationTypeRector extends AbstractRector implements MinPhpVersionInterface
+final class TypedPropertyFromToOneRelationTypeRector extends AbstractRector implements MinPhpVersionInterface, ConfigurableRectorInterface
 {
+    public const FORCE_NULLABLE = 'force_nullable';
+
+    private bool $forceNullable = true;
```

Comment:
Default value should be `false`, not to change current behavior and create BC break.